cordova-plugin-appcenter-push/src/android/AppCenterPush.gradle

```diff
@@ -1,9 +1,15 @@
 buildscript {
     repositories {
+        google()
         jcenter()
         mavenLocal()
     }
     dependencies {
         classpath 'com.android.tools.build:gradle:+'
+        classpath 'com.google.gms:google-services:4.0.1'
     }
-}
\ No newline at end of file
+}
+
+ext.postBuildExtras = {	
```

Comment:
We will tell people in docs to download google-services.json and put it in platforms/android/app folder. The resource file directive is not worth it as its just a copy.